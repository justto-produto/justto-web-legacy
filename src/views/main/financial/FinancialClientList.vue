<template>
  <jus-view-main />
</template>

<script>
export default {
  name: 'FinancialCLientList',
}
</script>

<style>

</style>
