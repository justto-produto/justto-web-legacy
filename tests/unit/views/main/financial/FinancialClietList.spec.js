import FinancialClientList from '@/views/main/financial/FInancialClientList'
import { shallowMount } from '@vue/test-utils'

describe('Views > FinancialClientList', () => {
  const wrapper = shallowMount(FinancialClientList)

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
