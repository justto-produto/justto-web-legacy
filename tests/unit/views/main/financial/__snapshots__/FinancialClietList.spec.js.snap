// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Views > FinancialClientList renders correctly 1`] = `<jus-view-main />`;
